res/lang/production/production_de.txt

```diff
@@ -149,6 +149,6 @@ ABBREVIATION_YEARS = J.
 STUDIO_MANAGER         = Studioleiter
 STUDIO_MANAGER_TOOLTIP = Gibt Informationen zu geplanten Produktionen und händigt Einkaufslisten aus
 
-THE_LICENCE_OF_X_IS_NOW_AT_YOUR_DISPOSAL = Die Lizenz "%TITLE%" steht Dir nun zur Verfügung.
-THE_LIVE_PRODUCTION_OF_X_JUST_FINISHED   = Die Liveproduktion "%TITLE%" wurde eben abgeschlossen.
+THE_LICENCE_OF_X_IS_NOW_AT_YOUR_DISPOSAL = Die Lizenz „%TITLE%“ steht Dir nun zur Verfügung.
```

Comment:
Wollen wir bei Filmtiteln generell die andere Fassung nutzen - wie bei:
Erster Trailer zu «%MOVIETITLE%» veröffentlicht. %DIRECTORNAME1%: „Bester Film des Jahres“.

Reply:
Ich wäre hier gegen die Vermischung der Anführungszeichen. Wenn man es konsequent für alle Sprachen macht, müsste man dann für Sprachen, für die die spitzen Klammern normal sind, auch andere nehmen.

Wie wäre es, in diesem Fall die Titel in allen Sprachen fett rendern, so wie es auch bei den Produktionen gemacht wird?